README.md

```diff
@@ -0,0 +1,31 @@
+# JUnit Plugin for Jenkins
+
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/junit.svg)](https://plugins.jenkins.io/junit)
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/junit-plugin.svg?label=release)](https://github.com/jenkinsci/junit-plugin/releases/latest)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/junit.svg?color=blue)](https://plugins.jenkins.io/junit)
+
+The JUnit plugin provides a publisher that consumes XML test reports generated during the builds and provides some graphical visualization of the historical test results 
+(see [JUnit graph](https://wiki.jenkins.io/display/JENKINS/JUnit+graph) for a sample) 
+as well as a web UI for viewing test reports, tracking failures, and so on. 
+Jenkins understands the JUnit test report XML format (which is also used by TestNG). 
+When this option is configured, Jenkins can provide useful information about test results, such as trends.
+
+The plugin also provides a generic API for other unit-test publisher plugins in Jenkins. This functionality was part of the Jenkins Core until it was split out to this plugin in version in 1.577.
+
+## Configuration
+
+The JUnit publisher is configured at the job level by adding a Publish JUnit test result report post build action. The configuration parameters include:
+
+* **Test report XMLs:** Specify the path to JUnit XML files in the Ant glob syntax, such as **/build/test-reports/*.xml. 
+  Be sure not to include any non-report files into this pattern. 
+  You can specify multiple patterns of files separated by commas. 
+  The base directory of the fileset is the workspace root.
+* **Retain long standard output/error:** If checked, any standard output or error from a test suite will be retained in the test results after the build completes. 
+  (This refers only to additional messages printed to console, not to a failure stack trace). 
+  Such output is always kept if the test failed, but by default lengthy output from passing tests is truncated to save space. 
+  Check this option if you need to see every log message from even passing tests, but beware that Jenkins's memory consumption can substantially increase as a result, even if you never look at the test results!
+* **Health report amplification factor:** The amplification factor to apply to test failures when computing the test result contribution to the build health score. 
```

Comment:
I think this sort of detailed help is best left in `help*.html` files in the product. Will just go stale here. The online docs should focus on higher-level guidelines and suggestions.

Reply:
... but this is just a port of the Wiki page, so fine for now.

Reply:
Yeah, did not want to spend much time rewording. Subject for contributions